bootstrap.sh

```diff
@@ -49,6 +49,18 @@ program_exists() {
     local ret='0'
     type $1 >/dev/null 2>&1 || { local ret='1'; }
```

Comment:
I know we were concerned about things being too bash(y) before... According to this SO answer `command -v` is the most POSIX compatible. I don't know if we care about that? I haven't used `command -v` before though.